src/components/QueryEditor/VisualQueryEditor/utils/operators.ts

```diff
@@ -0,0 +1,340 @@
+import { SelectableValue } from '@grafana/data';
+import { QueryEditorPropertyType } from 'schema/types';
+
+export type OperatorInfo = {
+  Operator: string;
+  Description: string;
+  Example?: string;
+  'Case-Sensitive'?: string;
+};
+
+export const toOperatorOptions = (
+  type: QueryEditorPropertyType = QueryEditorPropertyType.String
+): Array<SelectableValue<string>> => {
+  return OPERATORS[type].map((op) => ({
+    label: op.Operator,
+    value: op.Operator,
+    description: `${op['Case-Sensitive'] ? '(Case-Sensitive) ' : ''}${op.Description}`,
+    title: op.Example,
+  }));
+};
+
+export const isMulti = (operator?: string) => {
+  // Some operators can be used to compare multiple values
+  return !!operator && ['in', '!in', 'in~', '!in~', 'has_all', 'has_any'].includes(operator);
+};
+
+export const OPERATORS: { [type: string]: OperatorInfo[] } = {
```

Comment:
source of this is https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/datatypes-string-operators